pages/dkp/kommander/2.2/insights/insights-viewing/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle: Viewing insight items
+title: Viewing insight items
+excerpt: Viewing and troubleshooting insight items
```

Comment:
We can add a Troubleshooting subsection if we have time and some good use cases to share. Otherwise, I'll delete "troubleshoot" for now and we can revisit in r2.3.